### ISSUE TITLE
Data scraping in docker gives error with chromedriver

### ISSUE THREAD

**Issue by dapenghan** [acntech/fruit-classifier#29]
Some debugging information as following:

```
root@9121eab807e4:/fruit-classifier# python3 -m fruit_classifier.data_scraping
> /fruit-classifier/fruit_classifier/data_scraping/__main__.py(32)main()
-> {'keywords': keywords,
(Pdb) l
 27         keywords = ','.join(categories)
 28
 29         import pdb; pdb.set_trace()
 30
 31         arguments = \
 32  ->         {'keywords': keywords,
 33              'limit': limit,
 34              'print_urls': True,
 35              'usage_rights': 'labeled-for-nocommercial-reuse',
 36              'output_directory': str(destination_dir),
 37              'chromedriver': str(root_dir.joinpath('chromedriver'))}
(Pdb) p root_dir.joinpath('chromedriver')
PosixPath('/fruit-classifier/chromedriver')
(Pdb) p root_dir.joinpath('chromedriver').is_file()
True
(Pdb) n
...
(Pdb) n
(Pdb) pp Path(arguments['chromedriver']).is_file()
True
(Pdb) n


Item no.: 1 --> Item name = bananas
Evaluating...
Looks like we cannot locate the path the 'chromedriver' (use the '--chromedriver' argument to specify the path to the executable.) or google chrome browser is not installed on your machine (exception: Message: Service /fruit-classifier/chromedriver unexpectedly exited. Status code was: 127
)
SystemExit
```

**Comment by loeiten** [comment]
Wow, this was a more challenging issue than first anticipated 

**TL;DR**

Until this issue is resolved, I would recommend to scrape with your local machine and rather mount the local `generated_data` with the container `/fruit-classifier/generated_data`

---

So, first of all, it looks like google-chrome also needs to be installed (as the error message reads). Furthermore, it seems like the lack of display and possibly some port configuration is complicating the issue. After a couple of takes, I end up with the error message

```
Message: unknown error: Chrome failed to start: exited abnormally
```

Helpful message...thank you google.

Resources:

* [It looks like `libnss3` and possibly `libgconf2-4` needs to be installed (depending on chromedriver version)](https://gist.github.com/mikesmullin/2636776)
* [Docker for chromedriver server](https://github.com/RobCherry/docker-chromedriver)
* [Docker files with python and chromedriver (raises `Cannot assign requested address (99)` error)](https://github.com/joyzoursky/docker-python-chromedriver/blob/master/py3/py3.7/Dockerfile)
* [The `Cannot assign requested address (99)` error](https://github.com/RobCherry/docker-chromedriver/issues/15)
* [The `disconnected: unable to connect to renderer` error](https://github.com/SeleniumHQ/selenium/issues/4897)
* [Another take on a dockerfile](https://jpmelos.com/articles/how-use-chrome-selenium-inside-docker-container-running-python/)
* [And another](https://gist.github.com/varyonic/dea40abcf3dd891d204ef235c6e8dd79)
* [Yet another](https://documentation.codeship.com/pro/continuous-integration/browser-testing/)
